opensearch-service-domain-cdk/bin/app.ts

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
```

Comment:
Is this considered acceptable form for TS/CDK?

Reply:
Ah, I missed this.  AFAIK, * imports are evil in TypeScript just like other languages.  Can you please import the specific types you need?